spec/abstract/RISCV64/Init_A.thy

```diff
@@ -54,18 +62,25 @@ definition init_arch_state :: arch_state
      riscv_kernel_vspace = init_vspace_uses
    \<rparr>"
 
+definition toplevel_bits :: nat
+  where
+  "toplevel_bits = pt_bits_left max_pt_level"
+
+definition elf_index :: pt_index
+  where
```

Comment:
for this and prev definition, please move `where` to prev line

Reply:
No, please don't, that breaks the style of the rest of the file. We should restyle the entire file which currently is consistent or we should keep it consistent.

Reply:
Ah, my bad. Somehow missed the file this was in. As Gerwin said, ignore me.